sycl/include/sycl/detail/pi.h

```diff
@@ -1024,7 +1026,13 @@ using pi_image_desc = _pi_image_desc;
 
 typedef enum { PI_MEM_CONTEXT = 0x1106, PI_MEM_SIZE = 0x1102 } _pi_mem_info;
 
+typedef enum {
+  PI_PEER_ACCESS_SUPPORTED = 0x0,
+  PI_PEER_ATOMICS_SUPPORTED = 0x1
+} _pi_peer_attr;
```

Comment:
we need to define the returned value (and type) for these queries, is it pi_bool aka uint32_t?

Reply:
OK I forgot the to document the return type. I will do this now. Note that this is a comment so won't affect the test status of this PR that is all green (everything passing.)